src/Containers/Adapter/DockerCLI.php

```
@@ -0,0 +1,513 @@
+        $containerIds = [];
+
+
+        $output = '';
+
+        if (\count($containerIds) <= 0 && \count($filters) > 0) {
+            return []; // No containers found
+        }
+
+        $stats = [];
+
+        $containersString = '';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$containersString'. (undefined)

(UnusedLocalVariable)